[2024-07-27T14:43:07.678+0100] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-27T14:43:07.727+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: jobs_etl.Data_processing scheduled__2024-07-25T18:00:00+00:00 [queued]>
[2024-07-27T14:43:07.739+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: jobs_etl.Data_processing scheduled__2024-07-25T18:00:00+00:00 [queued]>
[2024-07-27T14:43:07.739+0100] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-27T14:43:07.755+0100] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): Data_processing> on 2024-07-25 18:00:00+00:00
[2024-07-27T14:43:07.767+0100] {standard_task_runner.py:63} INFO - Started process 5173 to run task
[2024-07-27T14:43:07.772+0100] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'jobs_etl', 'Data_processing', 'scheduled__2024-07-25T18:00:00+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/jobs_etl.py', '--cfg-path', '/tmp/tmposl75niq']
[2024-07-27T14:43:07.775+0100] {standard_task_runner.py:91} INFO - Job 224: Subtask Data_processing
[2024-07-27T14:43:07.879+0100] {task_command.py:426} INFO - Running <TaskInstance: jobs_etl.Data_processing scheduled__2024-07-25T18:00:00+00:00 [running]> on host huser
[2024-07-27T14:43:08.050+0100] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='hassounibarka@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='jobs_etl' AIRFLOW_CTX_TASK_ID='Data_processing' AIRFLOW_CTX_EXECUTION_DATE='2024-07-25T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-25T18:00:00+00:00'
[2024-07-27T14:43:08.052+0100] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-27T14:43:43.322+0100] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to success. Terminating instance.
[2024-07-27T14:43:43.323+0100] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-27T14:43:43.335+0100] {process_utils.py:132} INFO - Sending 15 to group 5173. PIDs of all processes in the group: [5173]
[2024-07-27T14:43:43.336+0100] {process_utils.py:87} INFO - Sending the signal 15 to group 5173
[2024-07-27T14:43:43.337+0100] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-07-27T14:43:43.337+0100] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-27T14:43:43.432+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=5173, status='terminated', exitcode=0, started='14:43:06') (5173) terminated with exit code 0
